### PR TITLE
make pagination link to the pagination rather than the first block

### PR DESCRIPTION
**Pull request by johnduffell**
When you use pagination on liveblogs, it will drop you at the pagination controls rather than the first block.  I've also fixed up the metadata in the head to link without the fragments.
When coming back to the first page, the original behaviour of dropping you at the top still applies.
comments welcome, and @OliverJAsh take a look!

### CODE REVIEW

File: article/app/views/fragments/liveBlogNavigation.scala.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
             </a>
```
Comment (OliverJAsh) [inline article/app/views/fragments/liveBlogNavigation.scala.html]: Do you need to add it to the above as well?

File: article/app/views/liveblog/liveBlogBody.scala.html [hunk 2]
```diff
@@ -81,8 +81,8 @@ <h1 itemprop="headline" class="content__headline js-score">@Html(article.trail.h
                                     </button>
                                 </div>
                             </div>
-                            @pageModel.pagination.map { pagination =>
-                                @if(!pagination.isNewestPage) {
+                            @if(pageModel.currentPage.isArchivePage) {
+                                @pageModel.pagination.map { pagination =>
```
Comment (OliverJAsh) [inline article/app/views/liveblog/liveBlogBody.scala.html]: You could get rid of this map because we do the same map in the fragment.

**Comment by OliverJAsh** [comment]
Nice :+1:

File: article/app/views/fragments/liveBlogNavigation.scala.html (hunk 1 above)
Comment (johnduffell) [inline article/app/views/fragments/liveBlogNavigation.scala.html]: as discussed this is always page 1 which should navigate to the top, as there's no nagivation available to snap down to

File: article/app/views/liveblog/liveBlogBody.scala.html (hunk 2 above)
Comment (johnduffell) [inline article/app/views/liveblog/liveBlogBody.scala.html]: good point, done!

**Pull request merged**